northstar/src/npk/manifest/mod.rs

```diff
@@ -42,7 +42,7 @@ pub struct Manifest {
     pub version: Version,
     /// Pass a console fd number in NORTHSTAR_CONSOLE
     #[serde(default, skip_serializing_if = "HashSet::is_empty")]
-    pub console: Console,
+    pub console: Permissions,
```

Comment:
Think this should be `ConsoleQoS`?

The rate limiting only makes sense for containers (not the externally accessible console connection).

Reply:
done!